scripts/delayed.js

```diff
@@ -1,7 +1,20 @@
 // eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './aem.js';
+import { loadScript, sampleRUM } from './aem.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
 // add more delayed functionality here
+async function loadAdobeDTM() {
+  await loadScript(`${window.hlx.codeBasePath}/scripts/jquery-3.7.1.min.js`);
+  const prod = 'https://assets.adobedtm.com/a441b904b50e/7a4facbbcffb/launch-039be8795dc8.min.js';
+  const stage = 'https://assets.adobedtm.com/a441b904b50e/7a4facbbcffb/launch-a2ae4c3b0523-staging.min.js';
+
+  if (window.location.host.endsWith('.page') || window.location.host.startsWith('localhost')) {
```

Comment:
Hi @andreituicu  - Shall we place this condition the other way? So that the else condition will always load stage and if condition would go for PROD.

Reply:
@VenkatKrishnaSN applied your suggestion. does this look good to you?